torchvision/prototype/models/optical_flow/raft.py

```diff
@@ -23,16 +21,16 @@
 
 
 class Raft_Large_Weights(WeightsEnum):
-    pass
-    # C_T_V1 = Weights(
-    #     # Chairs + Things
-    #     url="",
-    #     transforms=RaftEval,
-    #     meta={
-    #         "recipe": "",
-    #         "epe": -1234,
-    #     },
-    # )
+    C_T_V2 = Weights(
+        # Chairs + Things
+        url="https://download.pytorch.org/models/raft_large_C_T_V2-1bb1363a.pth",
+        transforms=RaftEval,
+        meta={
+            "recipe": "",  # TODO
+            "sintel_train_cleanpass_epe": 1.3822,
+            "sintel_train_finalpass_epe": 2.7161,
```

Comment:
I evaluated these with 1 GPU and batch-size = 1, which explains the very small difference with the ones reported in the README (1.3825 and 2.7148) which were evaluate with 8 GPUs and batch-size = 10.

The "most correct" ones are the ones reported here in the `meta` dict.

Reply:
Sounds good. Why not update the readme?